recipes/polyglot/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "polyglot" %}
+{% set version = "16.7.4" %}
+{% set sha256 = "f7d9cca9a212622548e9416fb89f1238b994b8860ef49e03b7c82c67f9b6269b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - cld2cffi_patch.patch  # fixes dependency issue with pyicu
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python >=3
+    - toolchain
```

Comment:
Do you need compilers here or something like that? If so, please add the compilers which you need using conda build 3. I would recommend if you need to compile something to replace ``toolchain`` to
```
- {{ compiler('c') }}
```